test/inductor/test_triton_kernels.py

```diff
@@ -1667,7 +1667,7 @@ def f(a, b):
         a = torch.randn(301, device=GPU_TYPE)
         b = torch.randn(301, device=GPU_TYPE)
 
-        backend = torch._dynamo.testing.AOTEagerAndRecordGraphs()
+        backend = torch._dynamo.testing.AotEagerAndRecordGraphs()
```

Comment:
I think, this is related to the clash between my PR (#137677) introducing the class name "AOT..." in the name and @anijain2305's (#138231) updating the class name to "Aot...". It's interesting that this test didn't fail in #138231's signals.

Reply:
Ah such a silly thing